### PR TITLE
FWD: rel/7.0 to rel/7.1

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Depends on https://github.com/scality/Arsenal/pull/360.

### CODE REVIEW

**Comment by electrachong** [comment]
@ironman-machine r+

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "aws-sdk": "2.28.0",
-    "arsenal": "scality/Arsenal#rel/7.1",
+    "arsenal": "scality/Arsenal#03f82ea89145e922bbe08621f19ac596a02235c6",
```
Comment (LaurenSpiegel) [inline package.json]: why?

Comment (bennettbuchanan) [reply package.json]: npm was not installing the latest changes in `rel/7.1` so I added the commit hash of  the latest merge commit for testing.

**Comment by nicolas2bert** [comment]
@ironman-machine r+

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
@ironman-machine try

**Pull request merged**